src/tests/cohort.integration.spec.ts

```diff
@@ -19,30 +19,15 @@ import {AuthenticationService} from '../app/services/authentication/authenticati
 import {AuthenticationServiceMock} from '../app/services/mocks/authentication.service.mock';
 import {SubjectSetConstraint} from '../app/models/constraint-models/subject-set-constraint';
 
-describe('Integration test for query saving and restoring', () => {
+describe('Integration test for cohort saving and restoring', () => {
 
-  // mocked query objects
+  // mocked cohort objects
```

Comment:
cohort definition no longer involves observation criteria and data table, thus removing them from tests